src/pages/DataSubmissionForm.js

```diff
@@ -14,7 +15,7 @@ export const DataSubmissionForm = () => {
   const onChange = ({ key, value, isValid }) => {
     /* eslint-disable no-console */
     console.log('StudyInfo OnChange:', key, value, isValid);
-    set(key, value, formData);
+    set(formData, key, value);
```

Comment:
Not sure why, but I couldn't get the lodash/fp version of set to actually update formData (but the regular lodash function seems to be working fine). I tested by adding `console.log(formData)` after the call to set, and with the lodash/fp version it always printed `{} `